docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -119,7 +119,7 @@ The following procedure describes how to install certificates on the database no
 
```

Comment:
typo in the link "How to Use Platform-Generate Certificates to Enable TLS" above ->Platform-Generated

Reply:
Where? I don't see it

Reply:
line 106

Reply:
Fixed